### ISSUE TITLE
Crash during modpack loading

### ISSUE THREAD

**Issue by fartunrus** [walksanatora/hexsky#27]
While i try to play with Hexxy Skies, it's just crash. I use:
hexxy skies 1.0.8
paucal 0.6.0
hexcasting 0.11.2
valkyrien skies 2.3.0 beta 5
kotlin 4.11.0
Patchouli 84
Caelus 3.2.0
architectury 9.2.14
cloth 11.1.136
inline 1.0.2
Modloader: Forge 47.3.0, but i tried to use 47.1.3 and 47.3.12
Attached crash report
[crash-2024-12-24_22.06.18-client.txt](https://github.com/user-attachments/files/18241566/crash-2024-12-24_22.06.18-client.txt)

**Comment by diesaifa** [comment]
same for me, hexxy skies 1.0.7 version loads though without a crash

**Comment by NoElkaTheGod** [comment]
Same issue here. However, the issue seems to disappear when hextweaks is installed (which is weird, since it's not a dependency).

**Comment by walksanatora** [comment]
yeah I accidentally set it up so although it checks "is hextweaks loaded" since it imports the hextweaks class it crashes.
